tyr/servers/server.py

```diff
@@ -136,8 +158,9 @@ def configure(self):
         self.establish_route53_connection()
 
         if self.ami is None:
-            self.log.warn('No AMI provided')
-            self.ami = 'ami-8fcee4e5'
+            self.log.warn('No AMI provided, searching for latest one...')
+            self.ami = self.get_latest_ami(self.ami)
```

Comment:
While the idea of retrieving the latest AMI for a platform if one isn't specified sounds nice, we could run into issues, at least on the Linux side.

Amazon Linux AMI updates are also Amazon Linux OS updates and we've had previous OS updates which broke our provisioning process. It would be unfortunate if a new OS update came out and we automatically started using that without vetting that everything will continue to work without issues.

Feel free to leave the implementation of `get_latest_ami`, but I'd prefer to hardcode default AMI IDs for now, instead of automatically using the latest. It'd be useful to have a default AMI per-platform.